bentoml/_internal/bento/build_dev_bentoml_whl.py

```diff
@@ -32,35 +39,27 @@ def build_bentoml_editable_wheel(target_path: str) -> None:
     # Find bentoml module path
     (module_location,) = importlib.util.find_spec("bentoml").submodule_search_locations  # type: ignore # noqa
 
-    bentoml_setup_py = os.path.abspath(os.path.join(module_location, "..", "setup.py"))  # type: ignore
+    # PEP517-compatible
+    bentoml_pyproject = os.path.abspath(os.path.join(module_location, "..", "pyproject.toml"))  # type: ignore
 
     # this is for BentoML developer to create Service containing custom development
     # branches of BentoML library, it is True only when BentoML module is installed
     # in development mode via "pip install --editable ."
-    if os.path.isfile(bentoml_setup_py):
+    if os.path.isfile(bentoml_pyproject):
         logger.info(
-            "BentoML is installed in `editable` mode; building BentoML distribution with the local BentoML code base. The built wheel file will be included in the target bento."
+            "BentoML is installed in `editable` mode; building BentoML distribution with the local BentoML code base. The built wheel file will be included in the target bento. The wheel is also PEP517-compatible."
```

Comment:
```suggestion
            "BentoML is installed in `editable` mode; building BentoML distribution with the local BentoML code base. The built wheel file will be included in the target bento."
```
no need to log things most developers probably don't immediate understand